src/extension/content-script/webln.js

```diff
@@ -24,7 +24,12 @@ const weblnCalls = [
 // calls that can be executed when webln is not enabled for the current content page
 const disabledCalls = ["webln/enable"];
 
-let isEnabled = false; // store if webln is enabled for this content page
+function loadEnabledState() {
```

Comment:
why do we do this in the webln provider? doesn't it make sense to store enabled state for all providers?

Reply:
haven't extended this for all the providers. let's first decide the flow